packages/point-and-click/src/components/StepEditor.tsx

```diff
@@ -36,7 +37,7 @@ export const StepEditor: FC<{
   getParentBound: () => Bounding;
   onBackButtonClick?: () => void;
 }> = ({ step, setStep, onBackButtonClick, getParentBound }) => {
-  const [numberOfElementsFound, setNumberOfElementsFound] = useState(0);
+  const [currentSelector, setCurrentSelector] = useState<string>("");
```

Comment:
Since this is derived, can figure out the following way (with or without memo):

```ts
  const selector = useMemo(
    () =>
      step.trigger?.selector
        ? step.trigger.selector
        : step.trigger?.path && toSelector(step.trigger.path),
    [step.trigger]
  );

```